packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx

```diff
@@ -318,13 +302,13 @@ export default connect(
   }),
   (dispatch: Dispatch<any>, props) => ({
```

Comment:
propose to add a type for props
```
(dispatch: Dispatch<any>, props: Props) => ({
```
then you don't need to cast it to any 
```
props.onOk(tmpl)
```

Reply:
👍 [33de419](https://github.com/epam/ketcher/pull/1892/commits/33de419369cf7716a3b04f3155a58292cedce379)